packages/react-liveness/src/components/FaceLivenessDetector/shared/Toast.tsx

```diff
@@ -1,28 +1,36 @@
 import * as React from 'react';
-import { Flex, View, ViewProps } from '@aws-amplify/ui-react';
+import { Flex, View, ViewProps, useTheme } from '@aws-amplify/ui-react';
 
 import { LivenessClassNames } from '../types/classNames';
 
 interface ToastProps extends ViewProps {
   variation?: 'default' | 'primary' | 'error';
   size?: 'medium' | 'large';
   heading?: string;
+  isInitial?: boolean;
```

Comment:
What does `isInitial` mean?

Reply:
Indicates it's the toast used in the start screen which has a different styling

Reply:
Can we just pass the style down?